blockstack_client/rpc.py

```diff
@@ -1040,6 +1040,43 @@ def PUT_name_transfer( self, ses, path_info, name ):
         self._reply_json(resp, status_code=202)
         return
 
+    def POST_raw_zonefile( self, ses, path_info ):
+        """
+        Publish a zonefile which has *already* been announced.
+        Return 200 and {'status': True, 'servers': ...} on success
+        Return 401 when zonefile invalid with {'error': ...}
+        """
+        request_schema = {
```

Comment:
Zone files can be arbitrary data in practice.  Blockstack Core requires any uploaded zone file data to be base64-encoded, in order to accommodate this.  Should this endpoint also accept arbitrary zone file data?  Or is this endpoint only intended for "well-formed" zone files that are guaranteed to be ASCII (or at a minimum, guaranteed to be JSON-serializable)?

Reply:
Sure, this can support b64